newIDE/app/src/InstancesEditor/SelectedInstances.js

```diff
@@ -190,6 +190,33 @@ export default class SelectedInstances {
     );
   }
 
+  getSelectionAABB(): Rectangle {
+    const selectionAABB = new Rectangle();
+    const selection = this.instancesSelection.getSelectedInstances();
+    let instanceRect = new Rectangle();
+    for (var i = 0; i < selection.length; i++) {
+      if (this.selectedRectangles.length === i) {
+        const newRectangle = new PIXI.Graphics();
+        newRectangle.hitArea = new PIXI.Rectangle(0, 0, 0, 0);
+        this.selectedRectangles.push(newRectangle);
+        this.rectanglesContainer.addChild(newRectangle);
+      }
```

Comment:
You're sure we need that?

Reply:
(this looks like code for rendering only, that is done in `render`? If it's needed, we absolutely need a comment explaining why this side effect is necessary and why it would be dangerous not to do it.)

Reply:
Indeed, it was a bad copy/paste.